packages/tupaia-web/src/layout/UserMenu/UserMenu.tsx

```diff
@@ -21,6 +21,7 @@ const UserMenuContainer = styled.div<{
 }>`
   display: flex;
   align-items: center;
+  border: 1px white;
```

Comment:
Can we use this logic here? `${({ secondaryColor, theme }) => secondaryColor || theme.palette.text.primary}`